Resources/doc/config.rst

```diff
@@ -405,9 +405,10 @@ Otherwise you will get a *auth failed* exception.
             connections:
                 default:
                     server: "mongodb://localhost:27017"
-                    username: someuser
-                    password: somepass
-                    authSource: db_you_have_access_to
+                    options:
+                        username: someuser
+                        password: somepass
+                        db: db_you_have_access_to
```

Comment:
Hmm, the `authSource` doesn't seem to work for me ... unless I'm doing something wrong.
`Unrecognized option "authSource" under "doctrine_mongodb.connections.app.options"  `